fe/fe-core/src/main/java/com/starrocks/scheduler/mv/MVPCTRefreshListPartitioner.java

```diff
@@ -221,10 +222,34 @@ public Set<String> getMVPartitionsToRefresh(PartitionInfo mvPartitionInfo,
 
     @Override
     public Set<String> getMVPartitionNamesWithTTL(MaterializedView materializedView,
-                                                  String start, String end,
+                                                  MVRefreshParams mvRefreshParams,
                                                   int partitionTTLNumber,
                                                   boolean isAutoRefresh) {
         int autoRefreshPartitionsLimit = materializedView.getTableProperty().getAutoRefreshPartitionsLimit();
+
+        // if the user specifies the start and end ranges, only refresh the specified partitions
+        boolean isCompleteRefresh = mvRefreshParams.isCompleteRefresh();
+        if (!isCompleteRefresh) {
+            Set<PListCell> pListCells = mvRefreshParams.getListValues();
+            Map<String, PListCell> mvPartitions = materializedView.getListPartitionItems();
+            Set<String> result = Sets.newHashSet();
+            for (Map.Entry<String, PListCell> e : mvPartitions.entrySet()) {
+                PListCell mvListCell = e.getValue();
+                if (mvListCell.getItemSize() == 1) {
```

Comment:
here multi values means multi-column list partition or single-column-multple-value partition?

Reply:
`single-column-multple-values` case since mv only supports  one partition column for now.